test/transceivercontroller/SimulcastTransceiverController.test.ts

```diff
@@ -419,19 +419,22 @@ describe('SimulcastTransceiverController', () => {
 
       videosToReceive = new DefaultVideoStreamIdSet([7]);
       videoSubscriptions = tc.updateVideoTransceivers(videoStreamIndex, videosToReceive);
-      expect(videoSubscriptions).to.deep.equal([0, 7, 0]);
-      verifyTransceiverDirections(['recvonly', 'inactive']);
+      expect(videoSubscriptions).to.deep.equal([0, 0, 0, 7]);
+      verifyTransceiverDirections(['inactive', 'inactive', 'recvonly']);
       subackFrame = new SdkSubscribeAckFrame({
         tracks: [new SdkTrackMapping({ streamId: 7, trackLabel: 'v_7' })],
       });
       videoStreamIndex.integrateSubscribeAckFrame(subackFrame);
       setTransceiverStreamId(videoStreamIndex, videosToReceive);
+
       videosToReceive = new DefaultVideoStreamIdSet([7, 8]);
       videoSubscriptions = tc.updateVideoTransceivers(videoStreamIndex, videosToReceive);
-      expect(videoSubscriptions).to.deep.equal([0, 7, 8]);
-      verifyTransceiverDirections(['recvonly', 'recvonly']);
+      expect(videoSubscriptions).to.deep.equal([0, 0, 0, 7, 8]);
```

Comment:
Note this won't grow endlessly because the browser will clean up and remove inactive transceivers.